CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## v2.0.0 (x)
+
+#### :boom: Breaking Change
+* [#x](x) Remove watchman guard ([@gabrielcsapo](https://github.com/gabrielcsapo))
+
+#### Committers: 1
+- Gabriel Csapo ([@chancancode](https://github.com/gabrielcsapo))
```

Comment:
I don't think you need to populate this file by yourself. It's done via `release-it`, and is updated during publish.